neo4j/manifest.json

```diff
@@ -1,33 +1,34 @@
 {
-  "integration_id": "neo4j",
-  "maintainer": "help@neo4j.com",
+  "display_name": "Neo4j",
+  "maintainer": "neo4j-cloud@neotechnology.com",
   "manifest_version": "1.0.0",
   "name": "neo4j",
-  "display_name": "Neo4j",
-  "guid": "a85ec8bb-e677-4089-ae8f-d1705c340131",
-  "short_description": "Integration with Neo4j Enterprise to monitor server performance.",
-  "support": "contrib",
-  "public_title": "Datadog-Neo4j Integration",
+  "metric_prefix": "neo4j.",
+  "metric_to_check": "neo4j.page_cache_hits_total",
+  "creates_events": false,
+  "short_description": "Gathers Neo4j metrics",
+  "guid": "76b634c6-fcb7-419f-87b5-c2ef53b6a9ad",
+  "support": "neo4j",
   "supported_os": [
     "linux",
     "mac_os",
     "windows"
   ],
-  "creates_events": false,
+  "public_title": "Datadog-neo4j Integration",
```

Comment:
```suggestion
  "public_title": "Datadog-Neo4j Integration",
```

Reply:
I believe this has been changed as well.